test_app/src/config/httpd-canitrust.conf

```diff
@@ -204,9 +204,9 @@ Include "conf/extra/httpd-noexample.conf"
     Header always set Strict-Transport-Security "max-age=2147483647; includeSubDomains"
 
     SSLEngine on
-    SSLCertificateFile "/usr/local/apache2/certs/test-canitrust.com/alt-name.cert.pem"
-    SSLCertificateKeyFile "/usr/local/apache2/certs/test-canitrust.com/alt-name.privkey.pem"
-    SSLCertificateChainFile "/usr/local/apache2/certs/test-canitrust.com/alt-name.fullchain.pem"
+    SSLCertificateFile "/usr/local/apache2/certs/test-canitrust.com/sslsub1.test-canitrust.com.cert.pem"
+    SSLCertificateKeyFile "/usr/local/apache2/certs/test-canitrust.com/sslsub1.test-canitrust.com.privkey.pem"
+    SSLCertificateChainFile "/usr/local/apache2/certs/test-canitrust.com/sslsub1.test-canitrust.com.fullchain.pem"
```

Comment:
Currently, the test_app server will not start up because the certificates are missing.